app/src/main/java/de/xikolo/App.kt

```diff
@@ -69,6 +75,10 @@ class App : Application() {
         configureRealm()
         configureDefaultSettings()
         configureWebView()
+        // QUESTION Is this the right place for the dynamicShorcuts call on launch of app?
+        if (Feature.shortcuts()) {
+            configureShortcuts()
+        }
```

Comment:
In the best practices, it's said that one should "check the number of objects returned by getDynamicShortcuts() each time you launch your app and re-publish dynamic shortcuts as needed". Is this the right place for that?